### ISSUE TITLE
Can't do inference 

### ISSUE THREAD

**Issue by a-akram-98** [facebookresearch/detr#353]
I'm using my own dataset but i tried to infer the model with the pre-trained weights to a custom image but it doesn't give anything
this is my inference code:
       image = Image.open('/content/img2/training/image_2/000979.png')
        image = np.asarray(image)
        image = image.astype('float32')
        image = np.transpose(image,(2,0,1))
        x = torch.from_numpy(image)
        x = F.normalize(x, mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
        x = x.to(device)
        x = utils.nested_tensor_from_tensor_list([x])
        # print(x.shape)
        output = model_without_ddp(x)
        probas = output['pred_logits'].softmax(-1)[0, :, :-1]
        keep = probas.max(-1).values > 0.7

the image size is 1247*375 and has a car in it

**Comment by a-akram-98** [comment]
I notices that I has to do a Resize(800) to solve the problem but why 800 !?

**Comment by alcinos** [comment]
The model was trained with particular shapes and sizes at training time, and if you provide widely different ones at evaluation time, it is expected that the model will behave sub-optimally.

Your image seems to have a very wide aspect ratio, you could try to crop it for better performance.

I believe I have answered your question and as such I'm closing this. Feel free to reach out if you have further concerns.
